volttron/platform/instance_setup.py

```diff
@@ -76,7 +76,8 @@ def _load_config():
             config_opts[option] = parser.get('volttron', option)
 
 
-def _update_config_file(instance_name='volttron1'):
+def _update_config_file(instance_name=None):
+    _load_config()
```

Comment:
note get_home is already called within load_config as well as the path exists etc.

Reply:
get_home needs to be called in update method too so that it can create new config if one doesn't exist